ibis/expr/types/relations.py

```diff
@@ -104,8 +99,76 @@ def f(  # noqa: D417
     return f
 
 
+# TODO(kszucs): cover it with tests
```

Comment:
If we're covering it by other means, then I think it's fine to leave it as is unless you have specific cases you want to test that have no overlap with hitting this function through the user APIs.